src/installer/corehost/build.sh

```diff
@@ -1,282 +1,103 @@
-        __rid_plat=freebsd.$major_ver
-    fi
-
-    if [ $__linkPortable == 1 ]; then
-        if [ "$(uname -s)" == "Darwin" ]; then
-            __rid_plat="osx"
-        elif [ "$(uname -s)" == "FreeBSD" ]; then
-            __rid_plat="freebsd"
-        else
-            __rid_plat="linux"
-        fi
-    fi
-}
-
-usage()
-{
-    echo "Usage: $0 --rootdir <path> --configuration <configuration> --arch <Architecture> --hostver <Dotnet exe version> --apphostver <app host exe version> --fxrver <HostFxr library version> --policyver <HostPolicy library version> --commithash <Git commit hash> [--xcompiler <Cross C++ Compiler>]"
-    echo ""
-    echo "Options:"
-    echo "  --rootdir <path>                  Path to the root of the repository. Required."
-    echo "  --configuration <configuration>   Build configuration (Debug, Release)"
-    echo "  --arch <Architecture>             Target Architecture (x64, x86, arm, arm64, armel)"
-    echo "  --hostver <Dotnet host version>   Version of the dotnet executable"
-    echo "  --apphostver <app host version>   Version of the apphost executable"
-    echo "  --fxrver <HostFxr version>        Version of the hostfxr library"
-    echo "  --policyver <HostPolicy version>  Version of the hostpolicy library"
-    echo "  --commithash <Git commit hash>    Current commit hash of the repo at build time"
-    echo "  -portable                         Optional argument to build portable platform packages."
-    echo "  --cross                           Optional argument to signify cross compilation,"
-    echo "                                    and use ROOTFS_DIR environment variable to find rootfs."
-    echo "  --stripsymbols                    Optional argument to strip native symbols during the build"
-
-    exit 1
-}
+usage_list=("-hostver <Dotnet host version>: Version of the dotnet executable.")
+usage_list+=("-apphostver <app host version>: Version of the apphost executable.")
+usage_list+=("-fxrver <HostFxr version>: Version of the hostfxr library.")
+usage_list+=("-policyver <HostPolicy version>: Version of the hostpolicy library.")
+usage_list+=("-commithash <Git commit hash>: Current commit hash of the repo at build time.")
 
 set -e
 
-SOURCE="${BASH_SOURCE[0]}"
-while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
-  DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
-  SOURCE="$(readlink "$SOURCE")"
-  [[ "$SOURCE" != /* ]] && SOURCE="$DIR/$SOURCE" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
-done
-DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
-__build_arch=
+__scriptpath="$(cd "$(dirname "$0")"; pwd -P)"
+__RepoRootDir="$(cd "$__scriptpath"/../../..; pwd -P)"
+
+__BuildArch=x64
+__BuildOS=Linux
+__BuildType=Debug
+__CMakeArgs=""
+__Compiler=clang
+__CompilerMajorVersion=
+__CompilerMinorVersion=
+__CrossBuild=0
+__IsMSBuildOnNETCoreSupported=0
+__NumProc=1
```

Comment:
Why 1?

Reply:
`__NumProc=1` was copied from `src/libraries/Native/build-native.sh`. But coreclr/build.sh and coreclr/bulid-native.sh do not explicitly declare it (and let the commons file define it). I will remove it from here.